wca-states.md

```diff
@@ -3,10 +3,10 @@
 <version>Version: June 19, 2017
 
 
-The WCA recognizes a total of 199 independent countries.
+The WCA recognizes a total of 199 countries/regions:
 
 
-## <states-list>UN Member States:
+## <states-list>
```

Comment:
Looks like the build is still failing, unfortunately. I think that's because the regex here: https://github.com/thewca/wca-regulations-compiler/blob/6b6aee02de716bb2c7a1b1626200b61bcc364bee/wrc/parse/lexer.py#L77 is expecting there to be at least one character after `<states-list>`. I think there are two ways we can fix this:

1. Change the wca-regulations-compiler to not require anything after `<states-list>`. If we're going to that effort, then it probably doesn't make sense for it to require the `##` anymore, either.
2. Combine lines 6 and 9 so this looks like:

```
## <states-list> The WCA recognizes a total of 199 countries/regions:
```

Reply:
@viroulep, do you see any issues with changing the regs compiler as I've described in 1) above?

Reply:
Fixed in thewca/wca-regulations-compiler@3ebbd8fe8fec7c9167296f59b2677696fe61a954.